cmd/bootstrapper/main.go

```diff
@@ -24,6 +23,8 @@ var (
 
 func init() {
 	rootCmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
+	// Set up logger that sends logs to StdErr
+	// Info and warning level logs should be bubbled up to StdOut instead
```

Comment:
Please call out why here too.

Fix the spelling of logs in the commit message.

Reply:
Fixed. Also added comments before writing to stdout why that is being done